package.json

```diff
@@ -174,7 +174,7 @@
     "@redhat-cloud-services/frontend-components-utilities": "^3.3.11",
     "@redhat-cloud-services/host-inventory-client": "1.2.0",
     "@redhat-cloud-services/rbac-client": "1.2.0",
-    "@scalprum/core": "0.4.0",
+    "@scalprum/core": "^0.4.0",
```

Comment:
This has to be locked as the package is still not stable.

Reply:
@Hyperkid123  done!